frontend/src/components/Alerts/AlertsBanner.tsx

```diff
@@ -21,15 +21,27 @@ const Center = styled.div`
     align-items: center;
 `
 
+export enum AlertCategory {
+    ERROR,
+    WARNING,
+    SUCCESS,
```

Comment:
Will we ever need Success? Maybe Error, warning, info is a better set?

Reply:
Used success because the color green is called success in the tokens and in the future we might want to use blue banners and then it makes sense for those to be info